Task_TheGridProblem/index.html

```diff
@@ -1,11 +1,23 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Grid Problem</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
```

Comment:
Make sure the path to the CSS file (index.css) is correct and the file is located in the same directory as the HTML file.